docs/get-started/cardano-node/installing-cardano-node.md

```diff
@@ -1,122 +1,146 @@
 ---
 id: installing-cardano-node
-title: Installing cardano-node and cardano-cli from source
-sidebar_label: Installing cardano-node
+title: Getting cardano-node and cardano-cli
+sidebar_label: Getting cardano-node
 sidebar_position: 2
 description: This guide shows how to build and install the cardano-node and cardano-cli from the source-code for all major Operating Systems
 image: /img/og/og-getstarted-installing-cardano-node.png
 ---
 
-# Installing the node from source
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-The **latest** version of the node may be downloaded from the [cardano-node GitHub Releases](https://github.com/intersectmbo/cardano-node/releases) page.
 
-## Prerequisites
+# Getting `cardano-node`
 
-To set up your platform, you will need:
-
-
-| Network | CPU Cores | Free RAM | Free storage | OS for Pasive Node | OS for Stake pool |
-| :---: | :---: | :---: | :---: | :---: | :---: |
-| Mainnet | 2 | 24GB | 150GB of free storage (250GB recommended for future growth) | Linux / Windows** / MacOS | Linux |
-| Testnet | 2 | 4GB | 20GB | Linux / Windows** / MacOS | Linux |
+Binaries for the **latest** version of the node may be downloaded from the [cardano-node GitHub Releases](https://github.com/intersectmbo/cardano-node/releases) page.
 
-****Note** If you are building on Windows, we recommend using WSL2 under Windows 10 as this provides a development and execution environment that is very similar to Ubuntu.
+Alternatively, one can build `cardano-node` from source code locally.
 
+## Building from source
 
-## System dependencies installation
+The preferred way of building `cardano-node` is via Nix, but the node is
+buildable also using standard Haskell tools after setting up the building
+environment.
 
-To download the source code and build it, you need the following packages and tools on your Linux system:
+### Hardware requirements
 
-* the version control system `git`,
-* the `gcc` C-compiler,
-* C++ support for `gcc`,
-* developer libraries for the arbitrary precision library `gmp`,
-* developer libraries for the compression library `zlib`,
-* developer libraries for `systemd`,
-* developer libraries for `ncurses`,
-* `ncurses` compatibility libraries,
-* the Haskell build tool `cabal`,
-* the GHC Haskell compiler (version `9.6.7` or above).
+To set up your platform, you will need:
 
-In Redhat, Fedora, and Centos:
 
+| Network | CPU Cores | Free RAM | Free storage | OS for Pasive Node | OS for Stake pool |
+| :---: | :---: | :---: | :---: | :---: | :---: |
+| Mainnet | 2 | 24GB | 150GB of free storage (250GB recommended for future growth) | Linux / Windows / MacOS | Linux |
```

Comment:
150GB is potentially outdated, should perhaps bump it to 300GB (keeping some buffer)

Reply:
Probably, yeah